File: src/main/java/net/ndrei/teslacorelib/netsync/ITeslaCorePackets.java
package net.ndrei.teslacorelib.netsync;

import net.minecraftforge.fml.common.network.simpleimpl.IMessage;

/**
 * Created by CF on 2016-12-03.
 */
public interface ITeslaCorePackets {
    void send(IMessage message);
}

File: src/main/kotlin/net/ndrei/teslacorelib/netsync/ITeslaCorePackets.kt
package net.ndrei.teslacorelib.netsync

import net.minecraftforge.fml.common.network.simpleimpl.IMessage

/**
 * Created by CF on 2017-06-28.
 */
interface ITeslaCorePackets {
    fun send(message: IMessage)
    fun sendToServer(message: IMessage)
}